Tools/SandboxTest.ps1

```diff
@@ -83,7 +83,35 @@ New-Item $tempFolder -ItemType Directory -ErrorAction SilentlyContinue | Out-Nul
 $WebClient = New-Object System.Net.WebClient
 
 function Get-Release {
-  $releasesAPIResponse = Invoke-RestMethod 'https://api.github.com/repos/microsoft/winget-cli/releases?per_page=100'
+  # TODO: Further improvements
+  # Sending authorized requests with GitHub CLI (aka. `gh.exe`) is much more convenient than using `Invoke-RestMethod`.
+  # It would be better integrate this script with GitHub CLI, since developers do not need creating and configuring PAT manually.
+
+  # First query an endpoint to see if credentials are valid.
+  $isValidCredential = $false
+  $requestParameters = @{
+    Uri = 'https://api.github.com/octocat'
+  }
+  if ($env:GITHUB_TOKEN) {
+    $requestParameters.Add('Authentication', 'Bearer')
+    $requestParameters.Add('Token', $(ConvertTo-SecureString "$env:GITHUB_TOKEN" -AsPlainText))
+    $isValidCredential = ([Int16](Invoke-WebRequest @requestParameters).Headers["X-RateLimit-Limit"][0]) -eq 60
+  }
+
+  # Query version information of microsoft/winget-cli
+  $requestParameters = @{
+    Uri = 'https://api.github.com/repos/microsoft/winget-cli/releases?per_page=100'
+  }
+  if ($isValidCredential) {
```

Comment:
Your variable seems backwards. Based on the name, it would  be assumed that `true` means the credential *is* valid. Suggest either renaming the variable or switching the logic